.github/workflows/release_prep.sh

```diff
@@ -9,7 +9,7 @@ TAG=${GITHUB_REF_NAME}
 PREFIX="rules_oci-${TAG:1}"
 ARCHIVE="rules_oci-$TAG.tar.gz"
 git archive --format=tar --prefix=${PREFIX}/ ${TAG} | gzip > $ARCHIVE
-SHA=$(shasum -a 256 $ARCHIVE | awk '{print $1}')
+SHA=$(sha256sum $ARCHIVE | awk '{print $1}')
```

Comment:
can we revert this change?

Reply:
our CI runner has the shasum binary and i'd like to keep the diff small.

Reply:
Sure, done